### PR TITLE
Updated README with proper tag and link to Stackoverflow

### PR DESCRIPTION
**Pull request by abimarank**
## Purpose
Fixing tag and link on Stackoverflow for ballerina language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,4 +72,4 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
 
 * The ballerina-dev@googlegroups.com mailing list is for discussing code changes to the Ballerina project.
 * Chat live with us on our [Slack channel](https://ballerina-platform.slack.com/).
-* User support should be posted on Stack Overflow with the #ballerina tag.
+* Technical questions should be posted on Stack Overflow with the [#ballerinalang](https://stackoverflow.com/questions/tagged/ballerinalang) tag.
```
Comment (pubudu91) [inline README.md]: #ballerinalang -> #ballerina

Comment (hasithaa) [reply README.md]: Correct tag is #ballerinalang

Comment (abimarank) [reply README.md]: Fixed.

Comment (hasithaa) [reply README.md]: Hi Abimaran, 

Can you reword your commit message? "corrected" -> "correct"

**Pull request merged**